src/pool/pool.service.ts

```diff
@@ -0,0 +1,179 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Cron } from '@nestjs/schedule';
+import { InjectRepository } from '@nestjs/typeorm';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import ApolloClient from 'apollo-client';
+import { HttpLink } from 'apollo-link-http';
+
+import gql from 'graphql-tag';
+import { Repository } from 'typeorm';
+import fetch from 'cross-fetch';
+
+import { Pool } from './pool.entity';
+import Big from 'big.js';
+import { configService } from 'src/config.service';
```

Comment:
Please group external imports together.
Also, please use relative paths to avoid build errors that can manifest in some cases.